sycl/doc/developer/ContributeToDPCPP.md

```diff
@@ -62,6 +62,9 @@ end-to-end or SYCL-CTS tests.
 - Add a helpful comment describing what the test does at the beginning and
   other comments throughout the test as necessary.
 
+- All identifiers used in `llvm/sycl` headers and source files must contain at
```

Comment:
```suggestion
- All identifiers used in `llvm/sycl` headers files must contain at
```
As @steffenlarsen noted, there is no need to have this restriction for source files as they cannot conflict with users files by design.

Reply:
I think the same rule applies to source files, but for a different reason. Obviously user-defined macros do impact DPC++ library source files, but we should keep the coding style consistent across the whole project.
Even though it's not expressed in the rule, please, apply this rule to source files in practice.